dcbr-api/api/models.py

```diff
@@ -82,7 +82,7 @@ class Operator(models.Model):
     
 
     def __str__(self):
-        return "Reg ID: \t %s %s , %s" % (self.registration_Number, self.last_name, self.first_name)
+        return "Reg ID: \t %s %s , %s" % (self.regNum, self.last_name, self.first_name)
```

Comment:
We could leave the full spelling `registration_number` for consistency with the rest of the code, and clarity.

Reply:
ok